graphql-dgs/build.gradle.kts

```diff
@@ -35,6 +35,7 @@ dependencies {
     implementation("com.apollographql.federation:federation-graphql-java-support")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-jdk8")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-reactor")
```

Comment:
I think it's fine to add this new dependency.

Reply:
@gnoeley , could you also run the dependency locks please?
```
./gradlew gL sL
```

Reply:
That's added updates for both `kotlinx-coroutines-reactor` and `spring-cloud-dependencies` which I'm assuming is expected?